Lib/gftools/scripts/add_font.py

```diff
@@ -214,7 +213,12 @@ def _MakeMetadata(args, is_new):
             ):
                 metadata.primary_script = script
 
-    metadata.license = font_license
+    try:
+        font_license = fonts.LicenseFromPath(args.directory)
+        metadata.license = font_license
+    except ValueError:
+        print("WARNING: Could not determine license for %s" % args.directory)
+        print("Continuing without setting license.")
```

Comment:
Thoughts on setting it to OFL if this fails?

Reply:
For the use case I have in mind, we very much do not want this to default to OFL.